assets/js/base/context/providers/add-to-cart-form/form/submit/index.js

```diff
@@ -106,7 +106,10 @@ const FormSubmit = () => {
 									'woo-gutenberg-products-block'
 								),
 								{
-									id: 'dadd-to-cart',
```

Comment:
Sneaky dadd

Reply:
Yup, not sure how that snuck in #6159 - typo maybe? Hard to catch in the review too since it's one character.